virt/node_maintenance/virt-automatic-certificates.adoc

```diff
@@ -0,0 +1,9 @@
+[id="virt-automatic-certificates"]
+= Automatic renewal of TLS certificates
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-automatic-certificates
+toc::[]
+
+include::modules/virt-automatic-certificates-renewal.adoc[leveloffset=+1]
+include::modules/virt-automatic-certificates-frequency.adoc[leveloffset=+2]
```

Comment:
As mentioned earlier, I think you should remove this second module and incorporate the information into the first module.

Reply:
Second module was removed and content moved into the first module.